src/containers/SearchPage/components/form/SearchConceptForm.jsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectT } from '@ndla/i18n';
+import Button from '@ndla/button';
+import { css } from '@emotion/core';
+import { getResourceLanguages } from '../../../../util/resourceHelpers';
+import ObjectSelector from '../../../../components/ObjectSelector';
+import { searchFormClasses } from './SearchForm';
+
+class SearchConceptForm extends Component {
+  constructor(props) {
+    super(props);
+
+    const { searchObject } = props;
+
+    this.handleSearch = this.handleSearch.bind(this);
+    this.emptySearch = this.emptySearch.bind(this);
+
+    this.state = {
+      search: {
+        query: searchObject.query || '',
+        language: searchObject.language || '',
+        types: 'concept',
+      },
+    };
+  }
+
+  onFieldChange = evt => {
+    const { value, name } = evt.target;
+    this.setState(
+      prevState => ({ search: { ...prevState.search, [name]: value } }),
+      this.handleSearch,
+    );
+  };
+
+  handleSearch(evt) {
+    if (evt) {
+      evt.preventDefault();
+    }
+    const { search } = this.props;
+    search({ ...this.state.search, page: 1 });
+  }
+
+  emptySearch(evt) {
+    this.setState(
+      { search: { query: '', language: '', types: 'concept' } },
+      () => this.handleSearch(evt),
+    );
+  }
+
+  render() {
+    const { t } = this.props;
+
+    return (
+      <form onSubmit={this.handleSearch} {...searchFormClasses()}>
+        <div {...searchFormClasses('field', '50-width')}>
+          <input
+            name="query"
+            placeholder={t('searchForm.types.conceptQuery')}
+            value={this.state.search.query}
```

Comment:
Gjerne hent ut search over her, typ const { search } = this.state; for bedre lesbarhet